resources/responses/skroutz/merchants_ecommerce_orders_show.yml

```diff
@@ -32,7 +32,7 @@
 "courier_voucher":null,"line_items":[{"id":"opaoe8M3pZ","shop_uid":"12456",
 "product_name":"Champion Hooded Sweatshirt M (212077-KK001)","quantity":1,
 "size":{"label":"Μέγεθος","value":"L","shop_value":"L"},"unit_price":23.99,
-"total_price":23.99}],"created_at":"2019-11-28T13:24:37+02:00",
+"total_price":23.99,"price_includes_vat":true}],"created_at":"2019-11-28T13:24:37+02:00",
```

Comment:
The example here is with `vat_exclusion_requested: true`. Let's set `price_includes_vat` value to `false`.

Reply:
As a matter of fact, we could remove `vat_exclusion_requested` details and set `price_includes_vat` to true, if we're to keep this product, or the other way around :)